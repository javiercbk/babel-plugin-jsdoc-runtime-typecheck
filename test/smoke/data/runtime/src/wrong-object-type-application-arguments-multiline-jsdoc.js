// @typecheck

/**
 * @param {Object} data
 * @param {Number} data.a
 * @param {Number} data.b
 * @returns {Number}
 */
function test(data) {
    return data.a + data.b;
}

test({
    a: 1,
    c: '2'
});
