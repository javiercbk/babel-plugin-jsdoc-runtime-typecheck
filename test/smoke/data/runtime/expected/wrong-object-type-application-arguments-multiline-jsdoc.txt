'TypeError: Parameter "data" in function "test" has wrong type. It\'s "{"a":1,"c":"2"}" now, but must have {{"a":"Number","b":"Number"}} type.'
